client/src/UnconfiguredDashboard.js

```diff
@@ -4,20 +4,21 @@ function UnconfiguredDashboard() {
   return (
     <div>
```

Comment:
Sorry one thing I forgot to check before, the layout of the unconfigured dashboard needs updating - can find an example in flight desktop.

this was something that came up in testing after we stopped working on it